### PR TITLE
Tur 54

### PR DESCRIPTION
**Pull request by hondarider29**
Backline code for the Calendar, also something is missing from the front end
frontend/src/pages/Home is missing some resource and I didn't touch the front end

### CODE REVIEW

**Comment by hondarider29** [comment]
the frontend issue was fixed with an extra command to install the missing components, updated the read me

**Comment by hondarider29** [comment]
the change to fix the front end issue is now in the code, changing the read me

File: backend/models/Calender.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        vendorID: {
+          type: Number
+        }
+      })],
+      required: true,
+      default: new Map()
+  }
+});
+
+module.exports = Calendar = mongoose.model("calendar", CalendarSchema);
```
Comment (joesobo) [inline backend/models/Calender.js]: add new lines at the end of files

**Review by joesobo — APPROVED** [review]

File: backend/models/Calender.js (hunk 1 above)
Comment (hondarider29) [reply backend/models/Calender.js]: change made

**Review by rramani1230 — APPROVED** [review]

**Pull request merged**